### DISCUSSION TITLE
Stand Up Wednesday April 25th

### DISCUSSION THREAD

**Discussion by CristianCazares** [Bugs-io/TC3001B#70] [Main Stand Ups]
```markdown
**What did you achieve in the last 24 hours?**:
- 
**What are your priorities for the next 24 hours?**:
- 
**Blockers**:
- 
**Shoutouts**:
- @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Worked a little bit with @Jacobgtz024 

**What are your priorities for the next 24 hours?**:
- Finish data analytics first endpoints so that we can deploy the service

**Blockers**:
- Been busy with work

**Shoutouts**:
- @BrandonMagana

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Set up the API Gateway
- Implemented a simple api placeholder

**What are your priorities for the next 24 hours?**:
- Get the backend running in my computer????

**Blockers**:
- Python is not establishing the connection to postgres

**Shoutouts**:
- @BrandonMagana and @CarlosCRG19 for helping out with linux stuff

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
- Upload file workflow is completed
- Fix postgres connection issue

**What are your priorities for the next 24 hours?**:
- Discuss overall implementation
- Implement error handling on uploading file workflow

**Blockers**:
- None

**Shoutouts**:
- @Davbot02
